pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -115,8 +115,15 @@ protected String getQuery(SlingHttpServletRequest request) {
 
         // Module type filter
         if(!Strings.isNullOrEmpty(type)) {
-            StringBuilder moduleTypeCondition = new StringBuilder()
-                    .append("*/*/*/*/metadata/@pant:moduleType = '" + type + "'");
+            StringBuilder moduleTypeCondition;
+            if (type.equalsIgnoreCase("assembly")) {
+                moduleTypeCondition = new StringBuilder()
```

Comment:
can we refactor/rename moduleType to contentType ?

Reply:
done